src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -146,17 +131,33 @@ C_FUNC(\Name\()_End):
 
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
+	// Make the stack operation compatible with GDB and LIBUNWIND
+	PROLOG_PUSH		"{r7, lr}"
```

Comment:
@jkotas `push {r7, lr}` does not break `TransitionBlock` because the contents for `TransitionBlock` is created AFTER this: LINE 152, `strm r6, {r8 - r11, lr}`.